lightwood/api/json_ai.py

```diff
@@ -7,8 +7,8 @@
                                  Output, ProblemDefinition)
 
 
-trainable_encoders = ('PretrainedLangEncoder', 'CategoricalAutoEncoder', 'TimeSeriesEncoder', 'TimeSeriesPlainEncoder')
-ts_encoders = ('TimeSeriesEncoder', 'TimeSeriesPlainEncoder', 'TsNumericEncoder')
+trainable_encoders = ('PretrainedLangEncoder', 'CategoricalAutoEncoder', 'TimeSeriesEncoder', 'ArrayEncoder')
```

Comment:
We need to figure out a better solution for this @paxcema @George3d6 but for now ok

Reply:
> The fact array is in the time-series encoder subsection is a little misleading; I recognize they might borrow very similar principles but it needs to be thought through more carefully.

That was a mistake on my end, I think it will behave perfectly if it's not part of those.